std_msgs/Header header

bool lanes_first_with_last_connected

pacsim/Landmark[] left_lane
pacsim/Landmark[] right_lane
pacsim/Landmark[] unknown

pacsim/Landmark[] time_keeping_gates